libqtile/core/manager.py

```diff
@@ -214,6 +210,9 @@ def root(self):
 
     @property
     def conn(self):
+        logger.warning(
+            'Qtile.conn is deprecated. Please use Qtile.core.conn instead.',
```

Comment:
Can we assert False in our tests some how to catch any internal uses?

Reply:
I can make a test fixture that patches this property to do this, maybe? It would be temporary I guess.

I think I got all the internal uses anyways...is it worth it?

Reply:
If you think you got all the internal users, I'd just remove this. I doubt people are using the xcffib connection directly in their configs.